### DISCUSSION TITLE
so-firewall error

### DISCUSSION THREAD

**Discussion by wojackkkk** [Security-Onion-Solutions/securityonion#12262] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

1tb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I've performed multiple installations of Security Onion, but I keep getting stuck at the same point. Even after adding hosts, it still asks me to add them through so-firewall. I've tried adding them, but it's still not working. What should I do?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/157751343/05333d4e-594d-4572-ba31-998ca08077a2)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by robbiemarshall** [comment]
Are you following the documentation for adding endpoints in the correct areas here:  https://docs.securityonion.net/en/2.4/firewall.html 

What type of endpoints are you trying to add?  Where are you trying to add them?
